src/backend/interfaces/REST/go.server/operations/notifications/notifications.go

```diff
@@ -0,0 +1,111 @@
+/*
+ * // Copyleft (ɔ) 2018 The Caliopen contributors.
+ * // Use of this source code is governed by a GNU AFFERO GENERAL PUBLIC
+ * // license (AGPL) that can be found in the LICENSE file.
+ */
+
+package notifications
+
+import (
+	"bytes"
+	. "github.com/CaliOpen/Caliopen/src/backend/defs/go-objects"
+	"github.com/CaliOpen/Caliopen/src/backend/interfaces/REST/go.server/middlewares"
+	"github.com/CaliOpen/Caliopen/src/backend/main/go.main"
+	"github.com/gin-gonic/gin"
+	swgErr "github.com/go-openapi/errors"
+	"net/http"
+	"strconv"
+	"time"
+)
+
+// GetPendingNotif handles GET /notifications
+// two optional params may be in query : `to` and `from` to narrow the notifications list to a time range.
+func GetPendingNotif(ctx *gin.Context) {
```

Comment:
RetrieveNotification is better for consistency naming

Reply:
I put "GET" verb within REST interface (GetMessage, GetDevice, GetContact), but "RETRIEVE" within store interface.